data/human/transport missions.txt

```diff
@@ -3225,6 +3225,7 @@ mission "Spacediving professionals"
 		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
 		not attributes "station"
 		not attributes "research"
+		not attributes "moon"
```

Comment:
The intent is indeed to exclude any planet with any of the given attributes. Both the 3-line as well as the 1-line version seems to give the intended selection, but changed to 1-line since that looks a little bit nicer.